gpu-operator/gpu-operator-rdma.rst

```diff
@@ -22,52 +22,80 @@ GPUDirect RDMA and GPUDirect Storage
 About GPUDirect RDMA and GPUDirect Storage
 ******************************************
 
-`GPUDirect RDMA <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html>`_ is a technology in NVIDIA GPUs that enables direct
+`GPUDirect RDMA <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html>`__ is a technology in NVIDIA GPUs that enables direct
 data exchange between GPUs and a third-party peer device using PCI Express. The third-party devices could be network interfaces
 such as NVIDIA ConnectX SmartNICs or BlueField DPUs, or video acquisition adapters.
 
-GPUDirect Storage (`GDS <https://docs.nvidia.com/gpudirect-storage/overview-guide/index.html>`_) enables a direct data path between local or remote storage, like NFS server or NVMe/NVMe over Fabric (NVMe-oF), and GPU memory.
-GDS leverages direct memory access (DMA) transfers between GPU memory and storage, which avoids a bounce buffer through the CPU. This direct path increases system bandwidth and decreases the latency and utilization load on the CPU.
+`GPUDirect Storage <https://docs.nvidia.com/gpudirect-storage/overview-guide/index.html>`__ enables a direct data path between local or remote storage, such as NFS servers or NVMe/NVMe over Fabric (NVMe-oF), and GPU memory.
+GDS performs direct memory access (DMA) transfers between GPU memory and storage.
+DMA avoids a bounce buffer through the CPU.
+This direct path increases system bandwidth and decreases the latency and utilization load on the CPU.
 
-To support GPUDirect RDMA, a userspace CUDA APIs and kernel mode drivers are required. Starting with
-`CUDA 11.4 and R470 drivers <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html#changes-in-cuda-11-4>`_, a
-new kernel module ``nvidia-peermem`` is included in the standard NVIDIA driver installers (e.g. ``.run``). The
-kernel module provides Mellanox Infiniband-based HCAs direct peer-to-peer read and write access to the GPU's memory.
+To support GPUDirect RDMA, userspace CUDA APIs are required.
+The kernel mode support is provided by one of two approaches: DMA-BUF from the Linux kernel or the legacy ``nvidia-peermem`` kernel module.
+NVIDIA recommends using the DMA-BUF rather than using the ``nvidia-peermem`` kernel module from the GPU Driver.
```

Comment:
Is there any documentation or official developer statement we can point readers to?

Reply:
Actually no. AFAIK, there's no public documentation about this work and I don't know the reason.